src/RegistrationController.js

```diff
@@ -198,6 +198,21 @@ export default BaseLoginController.extend({
         noCancelButton: true,
         title: loc('registration.form.title', 'login'),
         save: loc('registration.form.submit', 'login'),
+        modelEvents : { 'invalid' : 'modifyErrors' },
+        modifyErrors: function (model, errorResp) {
```

Comment:
@pradeepdewda-okta @sherizada-okta 
feels like we really should update Model props and validation function (https://github.com/okta/okta-ui/blob/master/packages/courage/src/framework/Model.js#L304)
or even change Courage (there was discussion on this but didn't find it now)

Reply:
@sherizada-okta We can create a refactor ticket for this to do it properly.

Reply:
Thank you!